src/gui/tray/ActivityItem.qml

```diff
@@ -15,6 +15,16 @@ MouseArea {
     property bool isChatActivity: model.objectType === "chat" || model.objectType === "room" || model.objectType === "call"
     property bool isTalkReplyPossible: model.conversationToken !== ""
 
+    property bool displayTalkReplyOptions: false
+    Connections {
+        target: activityModel
+        function onDisplayTalkReplyOptions(activityIndex) {
+            if (model.index === activityIndex) {
+                displayTalkReplyOptions = true;
+            }
+        }
+    }
+
```

Comment:
would it not be easier to use an extra role for activity items in the model ?

Reply:
I tried and it didn't work, I would need more time with it. I am sure I am doing something wrong.

Reply:
I figured out what I was doing wrong, please check my last commit @mgallien